buildroot/arm.config

```diff
@@ -1,5 +1,9 @@
 BR2_arm=y
 BR2_OPTIMIZE_3=y
+BR2_TOOLCHAIN_BUILDROOT_GLIBC=y
```

Comment:
Why are we forcing glibc here? (I assume that's the cause of the +1.93MB to the cpio)

Reply:
No particular reason, was just making it match everything else.

I suppose if we save space by switching to another libc, that's probably worth doing.

Reply:
Yeah using uClibc-ng on arm64 shrinks the cpio image by 35%: https://github.com/nathanchance/continuous-integration/commit/4fe34b885404493180a1ac565cadd86e0eeb963e

I'll push that change + reverting the arm32 image back to it as soon as Travis verifies everything works properly.